src/rpc/core_rpc_server.cpp

```diff
@@ -1742,6 +1771,23 @@ namespace cryptonote
     return true;
   }
   //------------------------------------------------------------------------------------------------------------------------------
+  bool core_rpc_server::on_get_net_stats_json(const COMMAND_RPC_GET_NET_STATS::request& req, COMMAND_RPC_GET_NET_STATS::response& res, epee::json_rpc::error& error_resp, const connection_context *ctx)
+  {
+    PERF_TIMER(on_get_net_stats);
+    // No bootstrap daemon check: Only ever get stats about local server
+    res.start_time = (uint64_t)m_core.get_start_time();
+    {
+      CRITICAL_REGION_LOCAL(epee::net_utils::network_throttle_manager::m_lock_get_global_throttle_in);
+      epee::net_utils::network_throttle_manager::get_global_throttle_in().get_stats(res.total_packets_in, res.total_bytes_in);
+    }
+    {
+      CRITICAL_REGION_LOCAL(epee::net_utils::network_throttle_manager::m_lock_get_global_throttle_out);
+      epee::net_utils::network_throttle_manager::get_global_throttle_out().get_stats(res.total_packets_out, res.total_bytes_out);
+    }
+    res.status = CORE_RPC_STATUS_OK;
+    return true;
+  }
+  //------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Why would you have two of them ?

Reply:
Not sure what you mean. Why I made both `on_get_net_stats_json` and `on_get_net_stats`? If yes, I just made both because I saw that other calls are duplicated in the same way.

Reply:
Which ones ? AFAIK it's only get_info (and I'm planning to merge them at some point).

Reply:
You are of course right. I jumped at a conclusion after seeing that single "get_info" example.

I deleted the second i.e. the "_json" method.